requirements.txt

```diff
@@ -1,74 +0,0 @@
-alembic==1.4.2
```

Comment:
We do not need to exacly remove this file, this file can still be _used_, especially with Docker! 

Accurate dependency resolution is a hard problem, and poetry does it really well, so it takes a toll on time required for doing `poetry install`, instead we can make sure to always [export our dependencies](https://python-poetry.org/docs/cli/#export) to a `requirements.txt` file, which would contain pinned versions and hash checks.

We can then re-use this file for people who absolutely still cannot use poetry for some reason, and we should also use this file for things like CI and deployment.

Reply:
Would simply auto-generating this file (during a e.g CI) be a good solution?

Reply:
No, the dev would generate and commit it (there are even git commit hooks for that) whenever the dev adds/updates/removes a package.

Reply:
But why? If it's the only purpose would be to be utilized by CI, then I think it's better to hide it.

Reply:
The only reason is because it is simpler, there are enough issues on the Poetry repo about the dependency resolution taking ages sometimes, because it's a costly operation.

Reply:
I was taking a shower when I realized that my question (above) is stupid and the reason is obvious. I agree totally with this.